packages/xo-server/docs/vm-sync-hook.md

```diff
@@ -0,0 +1,80 @@
+# VM Sync Hook
+
+> This feature is currently _unstable_ and might change or be removed in the future.
+>
+> Feedbacks are very welcome on the [project bugtracker](https://github.com/vatesfr/xen-orchestra/issues).
+
+Before snapshotting (with or without memory, ie checkpoint), XO can notify the VM via an HTTP request.
+
+A typical use case is to make sure the VM is in a consistent state during the snapshot process, for instance by making sure database writes are flushed to the disk.
+
+## Configuration
+
+The feature is opt-in via a tag on the VM: `xo:notify-on-snapshot`.
+
+By default, it will be an HTTP request on the port `1727`, on the first IP address reported by the VM.
```

Comment:
is it possible to use https ?

Reply:
It was my initial though but it added complexity (certificate generation and management) and I deemed it unnecessary because these calls don't contain sensitive data.

Reply:
that is especially important with a potential secret in the URL/header

Reply:
Agreed, but it's not the case in my proposal.